landing/src/landing.css

```
@@ -0,0 +1,201 @@
+
+@import "global.css";
+
+
+body  {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  -webkit-font-smoothing: antialiased;
+}/*페이지의 요소를 세로로 순차적으로 배치한다
+  폰트 픽셀 깨짐 현상 방지*/
+
+nav  {
+  display: flex;
+  justify-content: center;
+  position: sticky;
+  top: 0;
+  width: 100%;
+  background-color: #edf7ff;
+}/*nav 전체가 sticky로 움직인다*/
+
+.gnb  {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 94px;
+  padding: 20px 200px;
+  max-width: 120rem;
+  z-index: var(--gnb-layer);
+}/*gnb 요소 flex 가로 정렬 후 패딩 값 부여*/
+
+header  {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  background-color: #edf7ff;
+}/*header 요소 flex 세로 정렬*/
+
+.hero-header  {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2.5rem;
+  padding-top: 4rem;
+}
+
+.cta  {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 54px;
+  cursor: pointer;
+  background-image: linear-gradient(91deg, var(--primary) 0.12%, var(--skyblue) 101.84%);
+  border-radius: 8px;
+  color: var(--gray-light);
+  font-size: 18px;
+  font-weight: 600;
+}
+
+.cta-short  {
+  width: 128px;
+}
+
+.cta-long  {
+  width: 350px;
+}
+
+.slogan  {
+  text-align: center;
+  font-size: 4rem;
+  font-weight: 700;
+  line-height: 5rem;
+}
+
+.slogan-gradient  {
+  background-image: linear-gradient(91deg, var(--primary) 17.28%, var(--pink) 74.98%);
+}
+
+.background-clip-text  {
+  background-clip: text;
+  -webkit-background-clip: text;
+  color: transparent;
+}
+
+.hero-image  {
+  width: 75rem;
+  height: 36.875rem;
+}
+
+article  {
+  padding-top: 4.375rem;
+  padding-bottom: 7.5rem;
+  width: 100%;
+}
+
+section  {
+  display: grid;
+  justify-content: center;
+  column-gap: 10rem;
+  row-gap: 0.8rem;
+  width: 100%;
+  height: 550px;
+  padding: 50px 0;
+}
+
+section:nth-of-type(odd) {
+  grid-template:
+    ". image"
+    "title image"
+    "description image"
+    ". image"
+    /291px 55rem;
+}
+
+section:nth-of-type(even) {
+  grid-template:
+    "image ."
+    "image title"
+    "image description"
+    "image ."
+    /550px 29.1rem;
+}
+
+.title  {
+  grid-area: title;
+  font-size: 3rem;
+  font-weight: 700;
+  line-height: 4rem;
+}
+
+
+
+.title1-gradient  {
```

👍🏻👍🏻👍🏻 너무 좋습니다.
조금 더 추가적으로 고쳐보자면 네이밍을 이렇게 하면 어떨까요?
```suggestion
.gradient-red-to-skyblue  {
```

그렇다면 이 클래스 네임은 꼭 title1에만 종속되지 않는 `class` 가 될 것 같습니다!